packages/specs/swagger/src/constants.ts

```
@@ -1,5 +1,6 @@
+import {getValue} from "@tsed/core";
 import {join} from "path";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)